src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  box-sizing: border-box;
+  height: $card-height;
+  width: $card-width;
```

Comment:
The variable `$card-width` is used for the `width` property. Ensure that this variable is defined elsewhere in your SCSS files. If it's not defined, you need to declare it with a valid value before using it.